README.md

```diff
@@ -360,7 +434,7 @@ func (dest *Options) UnmarshalJSON(buf []byte) error {
     result.MinDateMS = time.Now().UnixMilli() - 10000
  
     // Perform deserialization.
-    err := json.Unmarshal(result)
+    err := json.Unmarshal(buf, result)
     if err != nil {
         // TODO: Presumably, add some context.
         return err
```

Comment:
Apparently I am not allowed to comment on line 442, `return *result` seems to be wrong, as the return type is `error`.

Should `dest` be used inside that method instead of `result := new(...)` ?

Reply:
You're absolutely right, my bad!

I guess I've grown reliant on Rust's doctests, which actually compile & run your snippets.